.github/label-commenter-config.yml

```diff
@@ -0,0 +1,25 @@
+comment:
+  header: A brief note
+  footer: "\
+    ---\n\n
+    > &nbsp; &#9; &nbsp; &#9;  &nbsp; &#9;  &nbsp; &#9; Be sure to [join the community](http://slack.layer5.io), if you haven't yet and please leave a :star: [star on the project](../stargazers) :smile: on the project.
+    "
+
+labels:
+  - name: issue/design required
+    labeled:
+      issue:
+        body: This issue has been labeled with 'design-required'. Note that prior to commencing on implementation, a design specification needs to be created and reviewed for approval. See [Creating a Functional Specification](https://docs.google.com/document/d/1RP3IWLc-MiQS-QYasqCoVuCH7--G87p5ezE5f_nOzB8/edit?usp=sharing) to create a design spec.
+        action: open
+  - name: issue/remind
+    labeled:
+      issue:
+        body: Checking in... it has been awhile since we&#39;ve heard from you on this issue. Are you still working on it? Please let us know and please don&#39;t hesitate to contact a [MeshMate](https://layer5.io/community/meshmates/) or any other [community member](https://layer5.io/community/members) for assistance.
+        action: open
```

Comment:
Please add these steps https://github.com/sayantan1413/meshery-kuma/blob/master/.github/label-commenter-config.yml#L19-L21 here so that the github-action runs on the PR whenever we mark a issue/remind label.

A sample example of this workflow is here https://github.com/sayantan1413/meshery-kuma/pull/4

Reply:
Nice work :raised_hands: